packages/netlify-cms-core/src/backend.js

```diff
@@ -385,11 +385,17 @@ class Backend {
     const errors = [];
     const collectionEntriesRequests = collections
       .map(async collection => {
+        let summaryString = collection.get('summary') || '';
+        let results = summaryString ? summaryString.match(/(?<=\{\{)[^}]+(?=\}\})/g) : [];
```

Comment:
Chrome is the only browser that supports negative lookbehinds, we'll need a different approach for this.